great_expectations/rule_based_profiler/estimators/quantiles_numeric_range_estimator.py

```diff
@@ -22,10 +25,13 @@
 from great_expectations.types.attributes import Attributes
 from great_expectations.util import convert_ndarray_to_datetime_dtype_best_effort
 
+if TYPE_CHECKING:
+    import numpy.typing as npt
+
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
-DEFAULT_QUANTILES_QUANTILE_STATISTIC_INTERPOLATION_METHOD = "nearest"
+DEFAULT_QUANTILES_QUANTILE_STATISTIC_INTERPOLATION_METHOD: Final[str] = "nearest"
```

Comment:
@Kilo59 Is `Final` in `Python`?  Sorry, I did not see it imported or defined.  Thank you.

Reply:
Yes, it's imported above from `typing_extensions`.
https://mypy.readthedocs.io/en/stable/final_attrs.html?highlight=Final#final-names-methods-and-classes

https://docs.python.org/3/library/typing.html#typing.Final